docs/ivshmem.md

```diff
@@ -0,0 +1,119 @@
+# Inter-VM shared memory device
+
+The Inter-VM shared memory device (ivshmem) is designed to share a memory region between a guest and the host.
+In order for all guests to be able to pick up the shared memory area, it is modeled by QEMU as a PCI device exposing
+said memory to the guest as a PCI BAR.
+
+Device Specification is at https://www.qemu.org/docs/master/specs/ivshmem-spec.html.
+
+Now we support setting a backend file to share data between host and guest.
+
+## Usage
+
+`--ivshmem`, an optional argument, can be passed to enable ivshmem device.
+This argument takes a file as a `path` value and a file size as a `size` value.
+```
+--ivshmem <ivshmem>  device backend file "path=</path/to/a/file>,size=<file_size/must=2^n>";
+```
+
+## Example
+
+Create a file with a size bigger than passed to CH:
```

Comment:
Please do not use CH as the abbreviation for Cloud Hypervisor.

Reply:
Replaced witch `cloud-hypervisor`.